docs/reference/setup/secure-settings.asciidoc

```diff
@@ -35,6 +35,8 @@ You will be prompted to enter the keystore password and the file `elasticsearch.
 
 NOTE: If you don't specify the `-p` flag or if you enter an empty password, the {es} keystore will be obfuscated but not password protected.
 
+Once the keystore is password-protected, Elasticsearch will prompt for a password at startup.
```

Comment:
Slight variation to avoid future tense and to make it clear it's not a one-time prompt:
```suggestion
When the keystore is password-protected, {es} prompts you to supply the password each time it starts.
```

Reply:
Good suggestions!